src/open_inwoner/scss/components/Sitemap/sitemap.scss

```diff
@@ -0,0 +1,5 @@
+.sitemap {
+  section {
```

Comment:
You actually made a correct `sitemap__section` this time, so why not target that...? (using `&__section`) No styles on tags please, but i can create a PR later and clean up these things myself if you promise to never do it again :-)